### DISCUSSION TITLE
react 打包

### DISCUSSION THREAD

**Discussion by wzw-beichen** [empjs/emp#244] [Q&A]
react 打包生成ts文件 ts高阶组件未生成ts相关引入

**Comment by ckken** [comment]
什么版本 emp

**Comment by wzw-beichen** [comment]
> 什么版本 emp

1.8.20

**Comment by wzw-beichen** [comment]
> 什么版本 emp

用const EditHoc = <T extends Record<string, any>>(WrappedComponent: React.ComponentType<T>) => { 
  class Edit extends Component {
  }
}
这种生成不了相关ts代码 
直接  class Edit extends Component {
  }这种组件是可以生成ts提示代码得

**Comment by ckken** [comment]
@flyyuan 可以验证下 2.0是否有这个问题
